srcpkgs/ncpamixer-git/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'ncpamixer-git'
+pkgname=ncpamixer-git
+version=20180705
+revision=1
+hostmakedepends="git cmake"
+makedepends="pulseaudio-devel ncurses-devel"
+short_desc="An ncurses mixer for PulseAudio"
+maintainer="Benjamín Albiñana <benalb@gmail.com>"
+license="MIT"
+homepage="https://github.com/fulhax/ncpamixer"
+provides="ncpamixer-${version}_${revision}"
+replaces="ncpamixer>=0"
+
+do_fetch() {
+	local url="git://github.com/fulhax/ncpamixer.git"
+	msg_normal "Fetching source from $url ...\n"
+	git clone ${url} ${pkgname}-${version}
+}
+	do_build() {
```

Comment:
don't indent these

Reply:
@benalb not possible to just use `build_style=gnu-makefile` ?

Reply:
It should be `build_style=cmake` and `build_wrksrc=src`

Reply:
This was the template I did: https://github.com/abenson/void-packages/blob/39ae19e6f360690275b946664dfc5adeed3746e0/srcpkgs/ncpamixer/template

Reply:
I took the ttyload-git template as base to make the ncpamixer one. 
Now, as this is my first PR, get the template suggested by abenson and update the PR or ...?

Reply:
Feel free to use my template. 

The `ttyload-git` template is really old. The current methods that seem to be preferred for Void is to preferably use tagged versions, or in rarer cases, use a git commit hash if the usptream doesn't tag properly. That doesn't seem to be an issue with this project, so it's recommended that the tag be used.

Reply:
Now, should I rename ncpamixer-git to ncpamixer, and then edit the template?

Reply:
Yes.